### DISCUSSION TITLE
Remap kitty_mod to Ctrl+A

### DISCUSSION THREAD

**Discussion by RedDocMD** [kovidgoyal/kitty#3457] [Q&A] [answered]
I used to use Alacritty + Tmux  until I discovered Kitty and I must say I am really impressed by its features. :smile: 

But my muscle memory sometimes bothers me, since my tmux mod key was `Ctrl+A` while in Kitty it is `Ctrl + Shift`. I tried setting `kitty_mod` to `Ctrl + A`, but then the terminal just goes nuts. Is there a way to properly do this?

**Comment by kovidgoyal** [comment, marked as answer]
kitty_mod has to be modifiers not keys. If you want to use multiple keypresses for shortcuts (though why you would want to torture your fingers is beyond me) then you will need to map individual actions in kitty.conf like this:

```
map ctrl+a>x close_window
```

which will let you press ctrl+a andthen x to close window and so on.

**Reply by RedDocMD** [reply]
Thanks! That helps. :+1:
